ddp-workspace/projects/toolkit-prion/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "toolkit-prion",
+  "version": "1.0.0",
+  "peerDependencies": {
+    "@angular/animations": "8.2.14",
+    "@angular/cdk": "8.2.3",
+    "@angular/common": "8.2.14",
+    "@angular/compiler": "8.2.14",
+    "@angular/core": "8.2.14",
+    "@angular/flex-layout": "8.0.0-beta.27",
+    "@angular/forms": "8.2.14",
+    "@angular/material": "8.2.3",
+    "@angular/platform-browser": "8.2.14",
+    "@angular/platform-browser-dynamic": "8.2.14",
+    "@angular/router": "8.2.14",
+    "@auth0/angular-jwt": "2.1.0",
+    "@nguniversal/express-engine": "8.1.1",
+    "@ngx-translate/core": "12.1.2",
+    "@ngx-translate/http-loader": "4.0.0",
+    "auth0-js": "9.13.2",
+    "auth0-lock": "11.21.0",
+    "classlist.js": "1.1.20150312",
+    "core-js": "2.5.4",
+    "ddp-sdk": "1.0.0",
+    "hammerjs": "2.0.8",
+    "kaop-ts": "3.1.2",
+    "ngx-cookie": "4.1.2",
+    "npm-sass": "2.2.1",
+    "package": "1.0.1",
+    "raw-loader": "0.5.1",
+    "rxjs": "6.5.4",
+    "tslib": "1.10.0",
+    "underscore": "1.9.1",
+    "web-animations-js": "github:angular/web-animations-js#release_pr208",
+    "zone.js": "0.9.1"
+  }
```

Comment:
After merge to `develop` we should make sure this are synced up with rest of repo (@fat23cat  recently upgraded project dependencies)